app/app.go

```diff
@@ -1842,7 +1842,21 @@ func (f *Filter) Query() bson.M {
 		query["teamowner"] = f.TeamOwner
 	}
 	if f.Platform != "" {
-		query["framework"] = f.Platform
+		parts := strings.SplitN(f.Platform, ":", 2)
```

Comment:
I think you can use `image.SplitImageName` here, there are some edge cases from what I remember.